### DISCUSSION TITLE
Please tell me where I can find a list of all vehicles and their numbers?

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#3067] [Q&A] [answered]
Something like these: vehicle = 2230595153 vehicle = 666166960, etc. I just want to know where I can find these numbers with human names and other information

**Comment by lonelybud** [comment, marked as answer]
https://wiki.rage.mp/index.php?title=Vehicles

**Comment by ghost** [comment]
> https://wiki.rage.mp/index.php?title=Vehicles

Thank u so much

**Comment by xesdoog** [comment]
Here's a text file:
[GTAV VehicleList.txt](https://github.com/YimMenu/YimMenu/files/15253112/GTAV.VehicleList.txt)

And this one is formatted as a Lua table:
[vehicleList.lua.txt](https://github.com/YimMenu/YimMenu/files/15253121/vehicleList.lua.txt)

**Reply by pierrelasse** [reply]
This is the vehicle list from a gta dumb formatted like `name=hash`
https://github.com/DurtyFree/gta-v-data-dumps/blob/master/VehicleList.ini
